blog-service/2024-03-27-search.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Index Aliasing for Search Queries
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - search
+  - partitions
+  - index
+  - aliasing
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to introduce the ability to index aliasing. With this update, you can now use an alias to point to one or more system indexes in the source expression of your search queries.
```

Comment:
```suggestion
We're excited to introduce the ability to create index aliasing. With this update, you can now use an alias to point to one or more system indexes in the source expression of your search queries.
```

Reply:
I don't think create would be the right word here, as user doesn't create anything on their end and rather references alias for system index in the query

Reply:
@atishya-22 , how about _We're excited to introduce index aliasing for Sumo Logic indexes._

Reply:
"Index aliasing" in itself doesn't seem intuitive enough i believe. 

We're excited to introduce ability to reference sumo logic indexes with an alias. 

@kimsauce  Something similar to this? ^^

Reply:
sounds good - I'll apply it

Reply:
@atishya-22 how about: _We're excited to introduce the ability to reference any Sumo Logic index with an alias._

Reply:
Sounds good. 
_We're excited to introduce the ability to reference Sumo Logic system indexes with an alias._

seems to give more precise idea?

Reply:
perfect!